src/models/OtrStore.js

```diff
@@ -0,0 +1,67 @@
+var _ = require("lodash");
+var DSA = require('otr').DSA;
+var OTR = require('otr').OTR;
+var Chan = require("./chan");
+
+module.exports = OtrStore;
+
+/** Stores OTR state for a given user
+ *
+ * Handels private key, current sessions and fingerprints.
+ */
+function OtrStore(name, client) {
+	_.merge(this, {
+		name: name,
+		privKey: undefined,
+		sessions: {},
+		client: client,
+	});
```

Comment:
Then, knowing that you deeply clone the `client` here, while you also deeply clone the `otrStore` [there](https://github.com/astorije/shout/pull/1/files#diff-cf27c1d543e886c89cd9ac8b8aeaf05bR66) and that, therefore, you allow yourself to a world of pain (`client.otrStore.client` and `client` references 2 different instances, same for `otrStore.client.otrStore` and `otrStore`...), you will have to live with my :-1:, do what you want with it :-).